lumicks/pylake/kymo.py

```diff
@@ -171,4 +171,6 @@ def save_tiff(self, filename, dtype=np.float32, clip=False):
             If enabled, the photon count data will be clipped to fit into the desired `dtype`.
             This option is disabled by default: an error will be raise if the data does not fit.
         """
-        save_tiff(self.rgb_image, filename, dtype, clip)
+        metadata = ImageMetaData.from_dataset(self.json)
+
+        save_tiff(self.rgb_image, filename, dtype, clip, metadata)
```

Comment:
Nitpick: you can just inline the `metadata` variable here.

Reply:
Done.